lib/bunny/session.rb

```diff
@@ -199,6 +201,7 @@ def initialize(connection_string_or_opts = ENV['RABBITMQ_URL'], optz = Hash.new)
 
       client_props         = opts[:properties] || opts[:client_properties] || {}
       @client_properties   = DEFAULT_CLIENT_PROPERTIES.merge(client_props)
+      @connection_name     = @client_properties.fetch(:connection_name, nil)
```

Comment:
`Hash#fetch` makes sense when you want a non-`nil` default. Should we use `Hash#[]` here?